lib/src/Base/Stat/openturns/CovarianceModel.hxx

```diff
@@ -72,19 +72,13 @@ public:
   virtual Scalar computeStandardRepresentative(const Scalar & s,
       const Scalar & t) const;
 
-  virtual Scalar computeStandardRepresentative(const Point & tau) const;
-
-  virtual Scalar computeStandardRepresentative(const Scalar & tau) const;
```

Comment:
Indeed, operator(), computeStandardRepresentative and computeAsScalar do not operate on Scalars, I'll fix that.